test/e2e/ha/command_test.go

```diff
@@ -0,0 +1,566 @@
+					{Name: "schematransformer", Image: "registry:5000/contrail-nightly/contrail-controller-config-schema:" + cemRelease},
+					{Name: "servicemonitor", Image: "registry:5000/contrail-nightly/contrail-controller-config-svcmonitor:" + cemRelease},
+					{Name: "analyticsapi", Image: "registry:5000/contrail-nightly/contrail-analytics-api:" + cemRelease},
+					{Name: "collector", Image: "registry:5000/contrail-nightly/contrail-analytics-collector:" + cemRelease},
+					{Name: "queryengine", Image: "registry:5000/contrail-nightly/contrail-analytics-query-engine:" + cemRelease},
+					{Name: "redis", Image: "registry:5000/common-docker-third-party/contrail/redis:4.0.2"},
+					{Name: "init", Image: "registry:5000/common-docker-third-party/contrail/python:3.8.2-alpine"},
+					{Name: "init2", Image: "registry:5000/common-docker-third-party/contrail/busybox:1.31"},
+					{Name: "statusmonitor", Image: "registry:5000/contrail-operator/engprod-269421/contrail-statusmonitor:" + scmBranch + "." + scmRevision},
+				},
+			},
+		},
+	}
+
+	command := &contrail.Command{
+		ObjectMeta: meta.ObjectMeta{
+			Namespace: namespace,
+			Name:      "command",
+			Labels:    map[string]string{"contrail_cluster": "command-ha"},
+		},
+		Spec: contrail.CommandSpec{
+			CommonConfiguration: contrail.PodConfiguration{
+				HostNetwork: &trueVal,
+			},
+			ServiceConfiguration: contrail.CommandConfiguration{
+				PostgresInstance:   "postgres",
+				KeystoneSecretName: "keystone-adminpass-secret",
+				KeystoneInstance:   "keystone",
+				SwiftInstance:      "swift",
+				ConfigInstance:     "config",
+				Containers: []*contrail.Container{
+					{Name: "init", Image: "registry:5000/contrail-nightly/contrail-command:" + cemRelease},
+					{Name: "api", Image: "registry:5000/contrail-nightly/contrail-command:" + cemRelease},
+					{Name: "wait-for-ready-conf", Image: "registry:5000/common-docker-third-party/contrail/busybox:1.31"},
+				},
+			},
+		},
+	}
+
+	return &contrail.Manager{
+		ObjectMeta: meta.ObjectMeta{
+			Name:      "command-ha",
+			Namespace: namespace,
+		},
+		Spec: contrail.ManagerSpec{
+			CommonConfiguration: contrail.ManagerConfiguration{
+				NodeSelector: map[string]string{nodeLabel: ""},
+				HostNetwork:  &trueVal,
+				Tolerations: []core.Toleration{
+					{
+						Effect:   core.TaintEffectNoSchedule,
+						Operator: core.TolerationOpExists,
+					},
+				},
+			},
+			KeystoneSecretName: "keystone-adminpass-secret",
+			Services: contrail.Services{
+				Memcached:  memcached,
+				Keystone:   keystone,
+				Postgres:   postgres,
+				Swift:      swift,
+				Command:    command,
+				Rabbitmq:   rabbitmq,
+				Cassandras: cassandra,
+				Zookeepers: zookeeper,
+				Config:     config,
+			},
+		},
+	}
+}
+
+func assertCommandSwiftContainerIsCreated(t *testing.T, proxy *kubeproxy.HTTPProxy, f *test.Framework, namespace string, adminPassword []byte) {
```

Comment:
I think that it is enough to check it in aio tests so it is unnecessary to check it in HA again

Reply:
Done